src/Common/TurbulenceClosures/TurbulenceClosures.jl

```diff
@@ -517,7 +595,7 @@ function turbulence_tensors(
     Richardson = diffusive.turbulence.N² / (normS^2 + eps(normS))
     f_b² = sqrt(clamp(1 - Richardson * _inv_Pr_turb, 0, 1))
 
-    β = (aux.turbulence.Δ)^2 * (α' * α)
+    β = f_b² * (aux.turbulence.Δ)^2 * (α' * α)
```

Comment:
Can we comment about this change in the PR description? This looks like a behavioral change (unless it’s somehow undone somewhere else).

Reply:
Ah ... I'll go through this again: this particular change is incorrect: It has been addressed in a prior PR. Seems I've committed an outdated file.

Reply:
@charleskawczynski Fixed in `799544`